src/messaging.h

```diff
@@ -151,6 +151,8 @@ class MessageEncoder {
   bool encode_bundles(SnapshotBundle system, SnapshotBundle application);
 #endif
 
+  int problematic_class_id() const { return problematic_class_id_; }
```

Comment:
Maybe it is time we let encode return a structured object (by value) instead? We have a number of different things we'd like to communicate.